packages/stockflux-launcher/src/titlebar/Titlebar.js

```diff
@@ -0,0 +1,68 @@
+import cx from 'classnames';
+import React, { useCallback } from 'react';
+import { OpenfinApiHelpers } from 'stockflux-core';
+import Components from 'stockflux-components';
+import './Titlebar.css';
+
+export default ({ dockedTo }) => {
+  const onMinimizeClick = useCallback(async () => {
+    const currentWindow = await OpenfinApiHelpers.getCurrentWindow();
+    currentWindow.minimize();
+  }, []);
+
+  const onCloseClick = useCallback(async () => {
+    const currentWindow = await OpenfinApiHelpers.getCurrentWindow();
+    currentWindow.close();
+  }, []);
+
+  const dockedHorizontal = dockedTo === 'top' || dockedTo === 'none';
```

Comment:
Should we not use `ScreenEdge` here? Also could be simplified by `[ScreenEdge.None, ScreenEdge.Top].includes(dockedTo)`;

Reply:
Also, I think this should use `titlebar` from stockflux-components rather than define it's own. The one in stockflux-components can be amended to accept JSX <p><strong>Stock</strong>Flux</p> as children.

Reply:
Raised https://github.com/ScottLogic/StockFlux/issues/1352